some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
+  "prBoard": "",
```

Comment:
/ticket remove this???

Reply:
Here's your ticket: [remove this???](https://ticketgen19.atlassian.net/browse/IKKI-34)